inc/admin.php

```diff
@@ -65,6 +65,10 @@ function ns_theme_check_admin_scripts( $hook ) {
 		return;
 	}
 	wp_enqueue_style( 'ns-theme-check-admin', NS_THEME_CHECK_URL . '/css/admin.css', array(), '0.1.3b' );
+	wp_enqueue_script( 'ns-theme-check-admin', NS_THEME_CHECK_URL . '/js/admin.js', array(), '0.1.3' );
+	wp_localize_script( 'ns-theme-check-admin', 'themeCheck', array(
```

Comment:
No need for localized object if it's only run on admin side. If you plan to run it on the front facing side, then sure.